charts/templates/_env.tpl

```diff
@@ -0,0 +1,120 @@
+{{/*
+Generate Tapir configuration with environment variables.
+*/}}
+{{- define "tapir.env" -}}
+{{- $config := .Values.deployment.configuration -}}
+{{- with $config.backend.type }}
+- name: BACKEND_CONFIG
+  value: {{ default "dynamodb" . }}
+{{- end }}
+{{- if (eq $config.backend.type "elasticsearch") }}
+{{- with $config.backend.elasticsearch.host }}
+- name: BACKEND_ELASTICSEARCH_HOST
+  value: {{ . }}
+{{- end }}
+{{- end }}
+{{- if (eq $config.backend.type "cosmosdb") }}
+{{- with $config.backend.cosmosDB.endpoint }}
+- name: BACKEND_AZURE_ENDPOINT
+  value: {{ . }}
+{{- end }}
+{{- with $config.backend.cosmosDB.masterKey }}
+- name: BACKEND_AZURE_MASTER_KEY
+  value: {{ . }}
+{{- end }}
+{{- end }}
+{{- with $config.storage.type }}
+- name: STORAGE_CONFIG
+  value: {{ default "s3" . }}
+{{- end }}
+{{- with $config.storageAccessDuration }}
+- name: STORAGE_ACCESS_SESSION_DURATION
+  value: {{ default 5 . }}
```

Comment:
```suggestion
  value: "{{ default 5 . }}"
```
we must ensure that the value is always string, otherwise the template does not render correctly